conferences/jsconf.yaml

```diff
@@ -32,9 +32,9 @@
 # JSConf US
 - jsconfus:
     name: JSConf US
-    site: https://2018.jsconf.us
+    site: https://2019.jsconf.us
     logo: https://2015.jsconf.us/img/js-sized.png
-    
+
```

Comment:
Consider adding `status: opencfp` and document that above :)

Reply:
Added `opencfp` status and documented in README.  I placed example in the jsconf.yaml file.